cloud/azure/versions.tf

```diff
@@ -10,7 +10,7 @@ terraform {
     }
     azuread = {
       source  = "hashicorp/azuread"
-      version = ">= 1.1"
+      version = ">= 2"
```

Comment:
may be it could be good to add providers requirements to the sfx/versions.tf too ?

Reply:
The sfx sub-module doesn't use azuread/azurerm providers